lighthouse-cli/bin.ts

```diff
@@ -114,6 +116,7 @@ Example: --output-path=./lighthouse-results.html`
   .default('output', Printer.GetValidOutputOptions()[Printer.OutputMode.pretty])
   .default('output-path', 'stdout')
   .default('port', 9222)
+  .default('timeout', 25)
```

Comment:
A bit overkill to bring in driver just for the constant to me. I don't think it's even that crazy that programmatic access and CLI access could have different sensible defaults at some point either, but I'm fine with removing if we like.